zkevm-circuits/src/bytecode_circuit/bytecode_unroller.rs

```diff
@@ -406,6 +406,7 @@ impl<F: Field> Config<F> {
         let length_is_zero_chip = IsZeroChip::construct(self.length_is_zero.clone());
 
         // Subtract the unusable rows from the size
+        debug_assert!(size > self.minimum_rows);
```

Comment:
The `bench` profile has `debug-assertions` and `overflow-checks` disabled, so in a benchmark this check will not be done (and neither the possible overflow in the next line).
I think we don't have code in the critical path in this repository, so I think we should use `assert!` instead of `debug_assert!` everywhere.  Only in critical functions (like some specific parts of the halo2 proving system side) should we avoid having `assert`.